Gemfile.lock.release

```diff
@@ -1204,9 +1223,8 @@ GEM
       rubocop-ast (>= 1.28.1, < 2.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 2.4.0, < 3.0)
-    rubocop-ast (1.44.1)
-      parser (>= 3.3.7.2)
-      prism (~> 1.4)
+    rubocop-ast (1.40.0)
```

Comment:
Interesting, why the regress in version number?

Reply:
https://github.com/ManageIQ/manageiq-style/pull/62

I'm fine with it.  Just wanted to check.

Reply:
This actually was the whole reason I did the manageiq-style thing in the first place, because newer versions of rubocop-ast don't work with older versions of rubocop.